src/mds/MDSRank.cc

```diff
@@ -295,7 +295,7 @@ void MDSRankDispatcher::tick()
   heartbeat_reset();
 
   if (beacon.is_laggy()) {
-    dout(5) << "tick bailing out since we seem laggy" << dendl;
+    dout(1) << "tick bailing out since we seem laggy" << dendl;
```

Comment:
while we're at this, mind making this a more informative message? :)

Reply:
Good suggestion, fixed!